s3_file_field/configuration.py

```diff
@@ -29,3 +31,11 @@ def get_storage_provider() -> StorageProvider:
         pass
 
     return StorageProvider.UNSUPPORTED
+
+
+def get_base_url() -> str:
+    # TODO: This could be memoized
```

Comment:
This can only be called after the app starts up, but will never change thereafter. Are there any suggestions on how to memoize without too much complexity?